tools/release/prepare-release.py

```diff
@@ -20,10 +20,12 @@ def make_version_select(root):
     with open(f'{root}/tools/release/templates/launchbuttons.html', 'r') as f:
         contents = f.read()
 
-    versions = ['0.12', '0.11', '0.10', '0.9', '0.8', '0.7', '0.6', '0.5', '0.4', '0.3']
+    versions = [
```

Comment:
If we start releasing all the time, what will we do when the drop-down menu is taller than the page? Will we start deleting old versions?